pkg/sql/plan/build_load.go

```diff
@@ -133,10 +133,10 @@ func buildLoad(stmt *tree.Load, ctx CompilerContext) (*Plan, error) {
 }
 
 func checkFileExist(param *tree.ExternParam) error {
-	param.Ctx = context.TODO()
 	if param.Local {
 		return nil
 	}
+	param.Ctx = context.TODO()
```

Comment:
param.Ctx uses the requestCtx instead of context.TODO

Reply:
done